app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormBranch.tsx

```diff
@@ -49,7 +47,6 @@ export const FormBranch = ({
   branch: AnnotatedBranch;
   equalRatio?: boolean;
   isReference?: boolean;
-  experimentFeatureConfig: getExperiment_experimentBySlug["featureConfig"];
```

Comment:
Also caught that individual branches don't need a reference to the selected feature config schema anymore, since all the bits of UI using it have since been removed.